### PR TITLE
l-n integrations: apply sentence-style capitalization

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Proposed change

l-n integrations: 
- apply sentence-style capitalization

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by gjohansson-ST — DISMISSED** [review]
LGTM 👍

File: source/_integrations/motion_blinds.markdown [hunk 1]
```diff
@@ -71,7 +71,7 @@ Please note that "-" characters need to be included in the key when providing it
 <img src='/images/integrations/motion_blinds/Motion_App__get_key_2.jpg' />
 </p>
 
-### Brel Home app
+### Brel home app
```
Comment (c0ffeeca7) [inline source/_integrations/motion_blinds.markdown]:
```suggestion
### Brel Home app
```

File: source/_integrations/knx.markdown [hunk 2]
```diff
@@ -129,7 +129,7 @@ Connection parameters are set up when adding the integration and can be changed
 
 Use `route back` if your tunneling server is located on a different network.
 
-### KNX Secure
+### KNX secure
```
Comment (c0ffeeca7) [inline source/_integrations/knx.markdown]:
```suggestion
### KNX Secure
```

**Review by gjohansson-ST — APPROVED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @c0ffeeca7 👍 

../Frenck

**Pull request merged**